api/tests/models/test_groups.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from api.models.devices import Device
+from api.models.groups import Group
+from asyncpg import UniqueViolationError
+
+
+class TestGroup:
+    group = Group(id=1, name="salon")
+    modified_group = Group(id=1, name="Salon")
+    group2 = Group(id=2, name="Cuisine")
+    device = Device(name="Lampe", modele="", type=1, ip="192.168.1.5")
+
+    @pytest.mark.asyncio
+    async def test_add(self, db):
+        assert self.group == await Group.add(self.group)
+
+        # Check that two groups cannot have the same name
+        with pytest.raises(UniqueViolationError):
+            await Group.add(self.group)
+
```

Comment:
Il serait judicieux d'aussi vérifier que l'info a été correctement enregistrée en DB.

```suggestion
        assert self.group == await Group.get(self.group.id)

```